package.json

```diff
@@ -1,8 +1,8 @@
 {
-  "name": "@adobe/asset-compute-sdk",
+  "name": "@nui/asset-compute-sdk",
   "description": "Adobe Asset Compute Worker SDK",
   "license": "Apache-2.0",
-  "version": "2.10.7",
+  "version": "0.0.1-pipeline-alpha",
```

Comment:
Once asset-compute-pipeline is open sourced sdk will need updates in package.json.